kubernetes/kubernetes.go

```diff
@@ -387,6 +388,10 @@ func (k KubernetesJobClient) Create(jobSpec *batchv1.JobSpec) (*batchv1.Job, err
 }
 
 func (k KubernetesJobClient) SetJobSchedule(schedule CronSchedule, jobSpec *batchv1.JobSpec) error {
+	successfulJobsHistoryLimit := helpers.GetEnvInt32("SUCCESSFUL_JOBS_HISTORY_LIMIT", 2)
+	failedJobsHistoryLimit := helpers.GetEnvInt32("FAILED_JOBS_HISTORY_LIMIT", 1)
+	concurrencyPolicy := getConcurrencyPolicy()
```

Comment:
I really wanted this to just be 

```
concurrencyPolicy := batchv1.ConcurrencyPolicy(helpers.GetEnv("CONCURRENCY_POLICY", "Forbid"))
```

 but I opted for the more safe way of doing it